data/schemas/validate_playerResponse.json

```diff
@@ -80,47 +94,97 @@
 							],
 							"pattern": "^.*$"
 						},
-						"pfphash": {
-							"$id": "#root/steamInfo/pfphash",
-							"title": "Pfphash",
+						"profileVisibility": {
+							"$id": "#root/steamInfo/profileVisibility",
+							"title": "ProfileVisibility",
 							"type": "string",
-							"default": "",
+							"default": "Private",
 							"examples": [
-								"f556979c2d55a633a063ff2b7b1eae1d2c2812fc"
+								"Public",
+								"Private"
 							],
 							"pattern": "^.*$"
 						},
-						"vacBans": {
-							"$id": "#root/steamInfo/vacBans",
-							"title": "Vacbans",
+						"pfpHash": {
+							"$id": "#root/steamInfo/pfpHash",
+							"title": "PfpHash",
 							"type": "string",
 							"default": "",
 							"examples": [
-								""
+								"f556979c2d55a633a063ff2b7b1eae1d2c2812fc"
 							],
 							"pattern": "^.*$"
 						},
-						"isFriend": {
-							"$id": "#root/steamInfo/isFriend",
-							"title": "Isfriend",
-							"type": "boolean",
+						"vacBans": {
+							"$id": "#root/steamInfo/vacBans",
+							"title": "VacBans",
+							"type": "integer",
+							"default": 0,
+							"examples": [
+								1
+							]
+						},
+						"gameBans": {
+							"$id": "#root/steamInfo/gameBans",
+							"title": "GameBans",
+							"type": "integer",
+							"default": 0,
 							"examples": [
-								true
-							],
-							"default": false
+								1
+							]
+						},
+						"daysSinceLastBan": {

```

Comment:
I've currently got it so `daysSinceLastBan` is null if they don't have any bans, do you want to change it to possibly be null or should I change it to return a value like 0 if there are no bans?

Reply:
I think having it be null would be more correct, however I don't object to just having a default value.

Reply:
bruh my terminal auto inserts backslashes around brackets help

Reply:
I think allowing NULL is a good idea. 0 days since last ban semantically means banned today, while NULL means never banned.